src/main/java/ru/skypro/homework/dto/CommentsDto.java

```diff
@@ -0,0 +1,17 @@
+package ru.skypro.homework.dto;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+@Builder
+public class CommentsDto {
+    private int count;
+    private List<CommentDto> results= new ArrayList<>();
+
+    public CommentsDto(int size, List<CommentDto> comments) {
+    }
```

Comment:
```suggestion
    public CommentsDto(int size, List<CommentDto> comments) {
        this.count = size;
        this.results = comments;
    }
```
Реализовываем конструктор